invenio_users_resources/records/api.py

```diff
@@ -102,21 +102,6 @@ class UserAggregate(Record):
 
     access = DictField("access")
 
-    _user = None
-    """The cached User entity."""
-
-    @property
```

Comment:
moved the cached actual model object into the mock model class, which makes more sense (the mock model having an actual model object backing it up, rather than the API class having a mock and an actual model object)